Readme.md

```diff
@@ -3,6 +3,28 @@
 [![Build Status](https://travis-ci.org/official-stockfish/Stockfish.svg?branch=master)](https://travis-ci.org/official-stockfish/Stockfish)
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/official-stockfish/Stockfish?svg=true)](https://ci.appveyor.com/project/mcostalba/stockfish)
 
+Multi Variant Stockfish is a [Stockfish](https://github.com/official-stockfish/Stockfish) fork specialized to play chess and some chess [variants](https://en.lichess.org/variant):
+- [Crazyhouse](https://en.lichess.org/variant/crazyhouse)
+- [Atomic](https://en.lichess.org/variant/atomic)
+- [Horde](https://en.lichess.org/variant/horde)
+- [King of the Hill](https://en.lichess.org/variant/kingOfTheHill)
+- [Racing Kings](https://en.lichess.org/variant/racingKings)
+- [Losers](https://en.lichess.org/variant/antichess)
```

Comment:
I suggest `Giveaway` rather than `Losers` as the caption here, unless I'm overlooking something.